gestalt/vault.py

```diff
@@ -12,6 +12,9 @@
 from retry.api import retry_call
 
 from gestalt.provider import Provider
+from dateutil.parser import isoparse
+
+EXPIRATION_THRESHOLD_HOURS = 1
```

Comment:
no need for hard coding this i think? you get the expiration time from the response.

Reply:
Yes, but we still need to calculate how much time is left. expire_time - current_time > EXPIRATION_THRESHOLD_HOURS.